linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -15,7 +15,10 @@ properties ([
         [$class: 'StringParameterDefinition',
             name: 'LIS_OLD_ARCHIVE_LINK',
             defaultValue: "${LIS_OLD_ARCHIVE_VERSION}",
-            description: 'Link to the previous lis version']
+            description: 'Link to the previous lis version'],
+        [$class: 'ChoiceParameterDefinition', choices: 'no\nyes',
+            name: 'FULL_RUN',
```

Comment:
this param is confusing as to what means full run or no full run
And the name should be FULL_PERF_RUN